src/stories/DividerWayFinder.stories.js

```diff
@@ -1,28 +1,91 @@
-import DividerWayFinder from '@/lib-components/DividerWayFinder.vue'
+import { computed } from 'vue'
+import DividerWayFinder from '@/lib-components/DividerWayFinder'
 
 export default {
   title: 'DIVIDER / Way Finder',
   component: DividerWayFinder,
 }
 
-export const Default = {
-  args: {},
+// MOCK DATA
+const MockVisit = {
+  color: 'visit',
 }
 
-export const Visit = {
-  args: {
-    color: 'visit',
-  },
+const MockAbout = {
+  color: 'about',
 }
 
-export const About = {
-  args: {
-    color: 'about',
-  },
+const MockHelp = {
+  color: 'help',
 }
 
-export const Help = {
-  args: {
-    color: 'help',
-  },
+// STORIES
+export function Default() {
+  return {
+    components: { DividerWayFinder },
+    template: '<divider-way-finder />',
+  }
+}
+
+export function Visit() {
+  return {
+    data() {
+      return {
+        ...MockVisit,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'visit'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
+  }
+}
+
+export function About() {
+  return {
+    data() {
+      return {
+        ...MockAbout,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'about'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
+  }
+}
+
+export function Help() {
+  return {
+    data() {
+      return {
+        ...MockHelp,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'help'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
+  }
+}
+
+export function FTVA() {
+  return {
+    provide() {
+      return {
+        theme: computed(() => 'ftva'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder />`,
+  }
```

Comment:
Add another story with router.push like in [blockRemoveSearchFilters](https://github.com/UCLALibrary/ucla-library-website-components/blob/vue3.x/src/stories/BlockRemoveSearchFilter.stories.js#L37) whenwe want to determine the color based on route and not based on color prop